examples/timeseries/event_classification_for_payment_card_fraud_detection.py

```diff
@@ -0,0 +1,430 @@
+
+We pre-process an event-set dataset into a tabular dataset and use a feed-forward neural network to learn the patterns of fraud and make predictions
+
+## Loading the dataset
+
+The dataset contains payment transactions sampled between April 1, 2018 and September 30, 2018. The transactions are stored in CSV files, one for each day.
+
+**Note:** Downloading the dataset takes ~1 minute.
+"""
+
+start_date = datetime.date(2018, 4, 1)
+end_date = datetime.date(2018, 9, 30)
+
+# Load the dataset as a Pandas dataframe.
+cache_path = "fraud_detection_cache.csv"
+if not os.path.exists(cache_path):
+    print("Download dataset")
+    dataframes = []
+    num_files = (end_date - start_date).days
+    counter = 0
+    while start_date <= end_date:
+        if counter % (num_files // 10) == 0:
+            print(f"[{100 * (counter+1) // num_files}%]", end="", flush=True)
+        print(".", end="", flush=True)
+        url = f"https://github.com/Fraud-Detection-Handbook/simulated-data-raw/raw/6e67dbd0a3bfe0d7ec33abc4bce5f37cd4ff0d6a/data/{start_date}.pkl"
+        dataframes.append(pd.read_pickle(url))
+        start_date += datetime.timedelta(days=1)
+        counter += 1
+    print("done", flush=True)
+    transactions_dataframe = pd.concat(dataframes)
+    transactions_dataframe.to_csv(cache_path, index=False)
+else:
+    print("Load dataset from cache")
+    transactions_dataframe = pd.read_csv(
+        cache_path, dtype={"CUSTOMER_ID": bytes, "TERMINAL_ID": bytes}
+    )
+
+print(f"Found {len(transactions_dataframe)} transactions")
+
+"""
+Each transaction is represented by a single row, with the following columns of interest:
+
+- **TX_DATETIME**: The date and time of the transaction.
+- **CUSTOMER_ID**: The unique identifier of the customer.
+- **TERMINAL_ID**: The identifier of the terminal where the transaction was made.
+- **TX_AMOUNT**: The amount of the transaction.
+- **TX_FRAUD**: Whether the transaction is fraudulent (1) or not (0).
+"""
+
+transactions_dataframe = transactions_dataframe[
+    ["TX_DATETIME", "CUSTOMER_ID", "TERMINAL_ID", "TX_AMOUNT", "TX_FRAUD"]
+]
+
+transactions_dataframe.head(4)
+
+"""
+The dataset is highly imbalanced, with the majority of transactions being legitimate.
+"""
+
+fraudulent_rate = transactions_dataframe["TX_FRAUD"].mean()
+print("Rate of fraudulent transactions:", fraudulent_rate)
+
+"""
+The [pandas dataframe](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) is converted into a [Temporian event-set](https://temporian.readthedocs.io/en/latest/reference/temporian/EventSet/), which is better suited for the data exploration and feature preprocessing of the next steps.
+"""
+
+transactions_eventset = tp.from_pandas(
```

Comment:
s/transactions_eventset/transactions_evset - let's do our best to keep all our docs/tutorials/learning material consistent with XXX_evset variable names and using EventSet when referring to an instance of the class